src/mapml/layers/FeatureLayer.js

```diff
@@ -5,7 +5,12 @@ export var FeatureLayer = L.FeatureGroup.extend({
    * M.MapML turns any MapML feature data into a Leaflet layer. Based on L.GeoJSON.
    */
     initialize: function (mapml, options) {
-
+      /*
+        mapml:
```

Comment:
Excellent work adding comments! It was a delight going over the code. Very helpful for new contributors or more importantly, for people revisiting the code and actually understand what's happening! 😄